intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -18,15 +18,19 @@ \chapter{Running batch jobs}
 nodes} of the cluster. There you can prepare the work you want to get done on
 the cluster by, e.g., installing or compiling programs, setting up data sets,
 etc. The computations however, should not be performed on this login node. The
-actual work is done on the cluster's \strong{compute nodes}. These compute
-nodes are managed by the job scheduling software (Moab) and a Resource Manager
-(TORQUE), which decides when and on which compute nodes the jobs can run. It is
-usually not necessary 
+actual work is done on the cluster's \strong{compute nodes}. Each compute node
+contains a number of CPU \strong{cores}. The compute nodes are managed by the
+job scheduling software (Moab) and a Resource Manager (TORQUE), which decides
+when and on which compute nodes the jobs can run. It is usually not necessary
 %% TODO: where?
 %(and in some cases not permitted)
 to log on to the
-compute nodes directly. Users can (and should) monitor their jobs
-periodically as they run, but do not have to remain logged in the entire
+compute nodes directly
+\ifbrussel
+and is only allowed on the node on which your job is running
```

Comment:
"on the nodes where you have a job running"

This is also true for (at least) Gent.

Reply:
ok, will add Gent